explanation/anbox-cloud.md

```diff
@@ -13,47 +17,61 @@ The Anbox Cloud Appliance is a self-contained, single-machine deployment variant
 
 The regular Anbox Cloud uses [Juju](https://juju.is/) for deployment and operations. It provides rich features and is ready made for a large scale deployment.
 
+See the following table for a comparison of features for the different variants:
+
+| Feature | Anbox Cloud Appliance | Anbox Cloud |
+|---------|-----------------------|-------------|
+| [Streaming capabilities](https://discourse.ubuntu.com/t/streaming-android-applications/17769) | ✓ | ✓ |
+| [Web dashboard](https://discourse.ubuntu.com/t/web-dashboard/20871) | ✓ | ✓ |
+| [Android API version](https://discourse.ubuntu.com/t/provided-images/24185) | 10, 11 | 10, 11 |
+| [Long term security updates](https://discourse.ubuntu.com/t/supported-versions/21046) | ✓ | ✓ |
```

Comment:
Do we provide that? The link says we don't ...

Reply:
For Anbox Cloud we don't but for Ubuntu we do. I think lets rephrase this as just "Security updates"

Reply:
I changed the link to https://ubuntu.com/support now.